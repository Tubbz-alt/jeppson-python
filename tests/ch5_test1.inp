7    6    2    10   0    0.001     0.00001217 0.1
8.0       12.0      10.0      12.0      10.0       6.0       8.0  
1106.0    751.0     1000.0    500.0     1200.0     600.0     800.0
0.0102    0.0102    0.0102    0.0102    0.0102     0.0102    0.0102
1    2     1    4
2000.0
0    3    -1   -2    5
1    3     2   -3   -7
-1500.0
1    2     3   -4
-1000.0
1    2    -5   -6
-1500.0
1    2     6    7
2000.0
4    1    -2   -3   -4
4    5    -6    7    2
