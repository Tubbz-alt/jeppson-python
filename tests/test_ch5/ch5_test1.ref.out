Pipe    Diameter         Length           Rel. Roughness
  0     8.0000E+00 in    1.1060E+03 ft    1.2750E-03
  1     1.2000E+01 in    7.5100E+02 ft    8.5000E-04
  2     1.0000E+01 in    1.0000E+03 ft    1.0200E-03
  3     1.2000E+01 in    5.0000E+02 ft    8.5000E-04
  4     1.0000E+01 in    1.2000E+03 ft    1.0200E-03
  5     6.0000E+00 in    6.0000E+02 ft    1.7000E-03
  6     8.0000E+00 in    8.0000E+02 ft    1.2750E-03


Iteration 0
Deviation 1.0000E+02 (Tolerance 1.0000E-03)

Pipe   Kp            expp          Qcurrent                  Qpredict
  0    2.0566E+00    2.1105E+00    3.9303E-01 ft ** 3 / s    3.9303E-01 ft ** 3 / s
  1    6.6798E-02    2.2303E+00    1.0743E-01 ft ** 3 / s    1.0743E-01 ft ** 3 / s
  2    2.4369E+00    2.0528E+00    1.8350E+00 ft ** 3 / s    1.8350E+00 ft ** 3 / s
  3    1.0144E+00    2.0372E+00    4.0630E+00 ft ** 3 / s    4.0630E+00 ft ** 3 / s
  4    9.0332E-01    2.1217E+00    5.0046E-01 ft ** 3 / s    5.0046E-01 ft ** 3 / s
  5    3.1426E+01    2.0150E+00    2.8416E+00 ft ** 3 / s    2.8416E+00 ft ** 3 / s
  6    5.4376E+00    2.0412E+00    1.6145E+00 ft ** 3 / s    1.6145E+00 ft ** 3 / s

Iteration 1
Deviation 3.2057E-01 (Tolerance 1.0000E-03)

Pipe   Kp            expp          Qcurrent                  Qpredict
  0    5.2540E+00    2.0556E+00    1.8162E+00 ft ** 3 / s    1.1046E+00 ft ** 3 / s
  1    2.1130E-01    2.1486E+00    8.0289E-01 ft ** 3 / s    4.5516E-01 ft ** 3 / s
  2    1.5464E+00    2.0753E+00    4.1182E-01 ft ** 3 / s    1.1234E+00 ft ** 3 / s
  3    8.4441E-01    2.0436E+00    2.6398E+00 ft ** 3 / s    3.3514E+00 ft ** 3 / s
  4    2.5121E+00    2.0596E+00    2.6191E+00 ft ** 3 / s    1.5598E+00 ft ** 3 / s
  5    1.9917E+01    2.0231E+00    7.2293E-01 ft ** 3 / s    1.7822E+00 ft ** 3 / s
  6    8.8279E+00    2.0268E+00    3.7331E+00 ft ** 3 / s    2.6738E+00 ft ** 3 / s

Iteration 2
Deviation 4.3933E-02 (Tolerance 1.0000E-03)

Pipe   Kp            expp          Qcurrent                  Qpredict
  0    4.9458E+00    2.0584E+00    9.6618E-01 ft ** 3 / s    1.0354E+00 ft ** 3 / s
  1    2.8567E-01    2.1268E+00    8.4303E-01 ft ** 3 / s    6.4910E-01 ft ** 3 / s
  2    1.6335E+00    2.0723E+00    1.2618E+00 ft ** 3 / s    1.1926E+00 ft ** 3 / s
  3    8.6096E-01    2.0429E+00    3.4898E+00 ft ** 3 / s    3.4206E+00 ft ** 3 / s
  4    2.6991E+00    2.0563E+00    1.8092E+00 ft ** 3 / s    1.6845E+00 ft ** 3 / s
  5    1.8561E+01    2.0246E+00    1.5328E+00 ft ** 3 / s    1.6575E+00 ft ** 3 / s
  6    9.2266E+00    2.0257E+00    2.9232E+00 ft ** 3 / s    2.7985E+00 ft ** 3 / s

Iteration 3
Deviation 1.3720E-03 (Tolerance 1.0000E-03)

Pipe   Kp            expp          Qcurrent                  Qpredict
  0    4.9342E+00    2.0585E+00    1.0302E+00 ft ** 3 / s    1.0328E+00 ft ** 3 / s
  1    2.8796E-01    2.1263E+00    6.6121E-01 ft ** 3 / s    6.5515E-01 ft ** 3 / s
  2    1.6367E+00    2.0722E+00    1.1978E+00 ft ** 3 / s    1.1952E+00 ft ** 3 / s
  3    8.6159E-01    2.0428E+00    3.4259E+00 ft ** 3 / s    3.4232E+00 ft ** 3 / s
  4    2.7042E+00    2.0562E+00    1.6914E+00 ft ** 3 / s    1.6879E+00 ft ** 3 / s
  5    1.8524E+01    2.0247E+00    1.6506E+00 ft ** 3 / s    1.6541E+00 ft ** 3 / s
  6    9.2376E+00    2.0257E+00    2.8054E+00 ft ** 3 / s    2.8019E+00 ft ** 3 / s

Iteration 4
Deviation 3.6329E-05 (Tolerance 1.0000E-03)

Pipe   Kp            expp          Qcurrent                  Qpredict
  0    4.9337E+00    2.0585E+00    1.0326E+00 ft ** 3 / s    1.0327E+00 ft ** 3 / s
  1    2.8802E-01    2.1263E+00    6.5547E-01 ft ** 3 / s    6.5531E-01 ft ** 3 / s
  2    1.6369E+00    2.0722E+00    1.1954E+00 ft ** 3 / s    1.1953E+00 ft ** 3 / s
  3    8.6161E-01    2.0428E+00    3.4234E+00 ft ** 3 / s    3.4233E+00 ft ** 3 / s
  4    2.7043E+00    2.0562E+00    1.6881E+00 ft ** 3 / s    1.6880E+00 ft ** 3 / s
  5    1.8523E+01    2.0247E+00    1.6540E+00 ft ** 3 / s    1.6540E+00 ft ** 3 / s
  6    9.2378E+00    2.0257E+00    2.8021E+00 ft ** 3 / s    2.8020E+00 ft ** 3 / s


Final results:

Pipe  Flow                     Flow                      Flow                    Head Loss       Head Loss
  0   2.9239E-02 m ** 3 / s    1.0326E+00 ft ** 3 / s    4.6345E+02 gal / min    1.5528E+00 m    5.0945E+00 ft
  1   1.8561E-02 m ** 3 / s    6.5547E-01 ft ** 3 / s    2.9420E+02 gal / min    5.7543E-02 m    1.8879E-01 ft
  2   3.3851E-02 m ** 3 / s    1.1954E+00 ft ** 3 / s    5.3655E+02 gal / min    5.9642E-01 m    1.9568E+00 ft
  3   9.6941E-02 m ** 3 / s    3.4234E+00 ft ** 3 / s    1.5365E+03 gal / min    8.9906E-01 m    2.9497E+00 ft
  4   4.7800E-02 m ** 3 / s    1.6881E+00 ft ** 3 / s    7.5765E+02 gal / min    1.3914E+00 m    4.5650E+00 ft
  5   4.6835E-02 m ** 3 / s    1.6540E+00 ft ** 3 / s    7.4235E+02 gal / min    9.3381E+00 m    3.0637E+01 ft
  6   7.9345E-02 m ** 3 / s    2.8021E+00 ft ** 3 / s    1.2577E+03 gal / min    7.8897E+00 m    2.5885E+01 ft

